eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -48,6 +48,9 @@ parameters:
           - eng/common/TestResources/sub-config/AzurePublicMsft.json
       Preview:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePaths: 
+          - eng/common/TestResources/sub-config/AzurePublicMsft.json
```

Comment:
This is not correct, as the subscription for preview is different. We will want to potentially add a new file for preview but for now it shouldn't be needed. I suggest removing the SubscriptionConfigurationFilePaths